worker.go

```diff
@@ -9,6 +9,10 @@ import (
 	"github.com/garyburd/redigo/redis"
 )
 
+// SleepBackoffsInMilliseconds is used to specify increasing times for a worker to wait when fetching jobs.
```

Comment:
If we expose the `var`, it becomes difficult to change, and would break backward compatibility in future. we could customise the newWorker when we initialise could pass in `config` or `functional options`.